snooty/test_postprocess.py

```diff
@@ -47,76 +53,91 @@ def test_deprecated_versions(backend: Backend) -> None:
     }
 
 
+def test_literal_includes(backend: Backend) -> None:
```

Comment:
Any reason this is still included here? Feel like we can handle all literal include testing in `test_parser.py` since the postprocessor isn't actually doing anything.

Reply:
Yeah, I've also considered deleting it. I've left it in since it's pretty concise, but it's up to you.

Reply:
I don't think it should be included since we don't touch literal includes in the postprocess layer!